tox_core/src/onion/client/mod.rs

```diff
@@ -169,7 +171,7 @@ struct OnionNode {
     /// Path used to send packets to this node.
     path_id: OnionPathId,
     /// Ping id that should be used to announce to this node.
-    ping_id: Option<sha256::Digest>,
+    ping_id: Option<[u8; <Sha256 as Digest>::OutputSize::USIZE]>,
```

Comment:
Would you please add an alias for ping_id type?

Reply:
Done.